headers/pmdsky.h

```diff
@@ -63,6 +63,14 @@ typedef unsigned short wchar_t;
 // whatever stdarg.h defines
 typedef void* va_list;
 
+// Define custom types for fixed-point numbers
+typedef int fx32_16;      // 32-bit signed fixed-point number with 16 fraction bits
```

Comment:
Can we move these to the same place as `struct fx64_16`? They ought to all live in the same place. Also, `pmdsky.h` is only for the core "system" types and things of that nature. I wouldn't consider these typedefs to fall in that category.

The compile-unsized action fails because this section is skipped in favor of importing the raw types from the real system headers (`stdint.h`, etc.). Which is related to the fact that this file only contains core system-level declarations.

Reply:
Sure, I'll place them in `util.h`